conf.py

```diff
@@ -59,7 +59,7 @@
     "webinars/contributing_to_pymc/index.md": "about/contributing_to_pymc/index.md",
     "webinars/intro_to_array_operations/index.md": "about/intro_to_array_operations/index.md",
     "webinars/probabilistic_programming_with_pymc/index.md": "about/probabilistic_programming_with_pymc/index.md",
-    "sprint/docstring_tutorial.md": "sprint/tutorials/docstring_tutorial.md",
+    "sprint/docstring_tutorial.md": "contributing/tutorials/docstring_tutorial.md",
     "index.md": "2023-03_sprint/index.md",
```

Comment:
```suggestion
    "index.md": "2023-03_sprint/index.md",
    "sprint/tutorials/docstring_tutorial.md": "contributing/tutorials/docstring_tutorial.md",
```

And this needs to be added, also for env setup and pr step by step